.gitignore

```
@@ -146,3 +146,6 @@ ajvReplacer.js
 
 # Need this due to shadcn default utils folder is lib
 !packages/shadcn/src/lib
+
+.nx/cache
+.nx/workspace-data
```

Adding a blank line at the end
```suggestion
.nx/workspace-data

```